### ISSUE TITLE
Update Travis' terraform version

### ISSUE THREAD

**Issue by raquelalegre** [UCL-CloudLabs/user-portal#61]
I think it's using 0.10 and there's 0.11 now. Also there might be a new way to install it with apt-get instead of the downloading the tarball.

**Comment by ageorgou** [comment]
Not 100% related to this, but a good place to note: new version of Terraform require the extra option `-auto-approve=true` when running `terraform apply`, so we should change the deployment command accordingly.

**Comment by raquelalegre** [comment]
Updated to 11.7 and waiting to see if Travis' tests work. - there seems to be an issue with Travis CI at the moment. New version seems to work locally.

**Comment by raquelalegre** [comment]
Somehow Travis tests always die at the same point after the update to the new version. Locally 11.7 works, but creating the Standard_A6 machine takes 11 minutes instead of under 2 minutes 🤷‍♀️ Not sure how this can be related to the version of Terraform though. Investigating...

**Comment by raquelalegre** [comment]
I'm not sure what happened. I've run a few local tests and they create machines under 2 mins on average. The Travis tests however kept on failing at the same point for a while:

```
      storage_image_reference.1458860473.offer:                           "UbuntuServer"
      storage_image_reference.1458860473.publisher:                       "Canonical"
      storage_image_reference.1458860473.sku:                             "16.04-LTS"
      storage_image_reference.1458860473.version:                         "latest"
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```
 
They seem to be OK now, so it must have been a combined Travis/Terraform/Azure glitch of some sort.

Closing.

**Comment by raquelalegre** [comment]
Oops, I should wait until merge to close this 😳

**Comment by ageorgou** [comment]
Closed in #72.
